packages/cli/__tests__/cli.test.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const cli = require('..');
```

Comment:
à quoit sert cette instruction?

Reply:
default file generated by lerna when created a package, we will update this anyway.

the `require('...')` is to import the lib and check everything exported by the library